### DISCUSSION TITLE
UAC1/2 in later cars?

### DISCUSSION THREAD

**Discussion by blerglemon** [BimmerGestalt/AAIdrive#348] [Q&A]
With Android Open Accessory Protocol 2 audio output unsupported in recent versions of Android I was wondering if it's replacement, UAC1 or UAC2 was the protocol used in newer cars? I tried finding that information elsewhere though I thought here would be a good place to ask. 
Cheers!

**Comment by hufman** [comment]
I also haven't heard any news about whether UAC is supported, even on IDrive 7. I'd be interested to hear if you find anything!
